MultiVersions/Forge/main/loginar/lang/zh_tw.json

```
@@ -0,0 +1,36 @@
+{
+  "block.loginar.huge_loginar_urn": "特大型洛琪納物品甕",
+  "block.loginar.large_loginar_urn": "大型洛琪納物品甕",
+  "block.loginar.medium_loginar_urn": "中型洛琪納物品甕",
+  "block.loginar.small_loginar_urn": "小型洛琪納物品甕",
+  "block.loginar.super_loginar_urn": "巨型洛琪納物品甕",
+  "block.loginar.tiny_loginar_urn": "迷你洛琪納物品甕",
+  "container.loginar.loginar_urn": "洛琪納物品甕",
+  "container.loginar.lunch_box": "午餐盒",
+  "entity.loginar.loginar": "洛琪納",
+  "item.loginar.backpack_upgrade": "背包化升級",
+  "item.loginar.backpack_upgrade.desc": "讓甕不用放置也能打開",
+  "item.loginar.item_swapper_upgrade": "物品交換升級",
+  "item.loginar.item_swapper_upgrade.desc": "按下綁定按鍵可以將單個物品和甕裡面的物品交換",
+  "item.loginar.loginar_antenna": "洛琪納觸鬚",
+  "item.loginar.loginar_antenna.lit": "洛琪納的觸角微微發亮",
+  "item.loginar.loginar_calamari": "烤洛琪納觸手",
+  "item.loginar.loginar_spawn_egg": "洛琪納 生怪蛋",
+  "item.loginar.loginar_tentacle": "洛琪納觸手",
+  "item.loginar.lunch_box": "午餐盒",
+  "item.loginar.lunch_box.desc": "Shift + 點擊右鍵打開以存放食物，按住右鍵直接食用裡面的食物。",
```

為了翻譯一致性，`Open` 習慣上是譯作 `開啟`
```suggestion
  "item.loginar.lunch_box.desc": "Shift + 點擊右鍵開啟以存放食物，按住右鍵則可以直接食用餐盒內的食物。",
```